daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -1492,7 +1492,7 @@ class EngineTest
             txMeta.submissionTime,
             now,
           )
-          .consume(lookupContract, lookupPackage, lookupKey)
+          .consume(lookupContractMap.lift, lookupPackage, lookupKey)
```

Comment:
i don’t understand why this changed.

Reply:
`lookupContract` is the default function for the whole file.
`lookupContractMap.lift` is the one that should be use in `lookup by key` tests.
The looked up contract is not in the former but in the latter. It was working before, because we did not need to fetch the contract for performing a lookup-by-key.